cigaradvisor/blocks/article-navigation/article-navigation.js

```diff
@@ -0,0 +1,50 @@
+import { getAllPosts } from '../../scripts/scripts.js';
+
+function getFinalOrCurrentUrl(url) {
```

Comment:
What's this needed for?

Reply:
article page will have a different url for imported articles , correct? which is 301 redirected to the sharepoint urls(present in the index) ?

Reply:
removed